src/web/server/locales/en/common.json

```diff
@@ -68,7 +68,7 @@
   "phoneNumber": {
     "title": "What’s your mobile telephone number?",
     "heading": "What’s your mobile telephone number?",
-    "hint": "This must be the same number you use for any benefits you claim.",
+    "formDescription": "If you claim any benefits you must use the same address registered with the benefits agency (DWP or HMRC).",
```

Comment:
Is this correct, should we be using the word `address` when this is on the phone number page?

Reply:
Well spotted. Chatted with Sam and 'address' has been updated with 'mobile number'. This will be added in the prototype shortly.